dna/zomes/projects/src/project/project_meta/validate.rs

```diff
@@ -25,6 +26,7 @@ pub fn validate_create_entry_project_meta(
 }
 
 #[hdk_extern]
+/// Updates allowed by anyone, but only `name` and `image` can be updated
```

Comment:
"but only `name` and `image` can be updated"

that used to be the case, but I reversed in a way to validate ones that can't change, as opposed to ones that can... so 
if project_meta.creator_address == original_project_meta.creator_address
                        && project_meta.created_at == original_project_meta.created_at
                        && project_meta.passphrase == original_project_meta.passphrase
creator_address, created_at, and passphrase can't change, the others can.

Reply:
```suggestion
/// Updates allowed by anyone, but only `name` and `image` can be updated whereas `creator_address`, `created_at`, and `passphrase` can't change
```

Reply:
There's a couple more fields that can change, that have been added. Just make the comment discuss the ones that can't be changed

Reply:
```suggestion
/// Updates allowed by anyone, but `creator_address`, `created_at`, and `passphrase` cannot be changed
```